release/models/lacp/openconfig-lacp.yang

```diff
@@ -407,6 +413,15 @@ grouping aggregation-lacp-members-statistics {
         system-id";
     }
 
+    leaf fallback {
+      type boolean;
+      description
+        "If the fallback is set to true, current LACP interface is
+        able to establish a Link Aggregation (LAG) before it receives
+        LACP PDUs from its peer, and fallback to a single port active
+        after the expiry of the timeout period.";
```

Comment:
IEEE or IETF references would be better

Reply:
I did some more searching and the fallback option does not appear to be documented in IEEE or other standards organization.  It is widely published as an option in device implementations.  I'll mark this resolved.